package.nls.json

```diff
@@ -119,6 +119,10 @@
     "AWS.command.ecr.createRepository": "Create Repository...",
     "AWS.command.ecr.deleteRepository": "Delete Repository...",
     "AWS.command.ecr.deleteTag": "Delete Tag...",
+    "AWS.command.emrserverless.startApplication": "Start EMR Serverless application",
+    "AWS.command.emrserverless.stopApplication": "Stop EMR Serverless application",
+    "AWS.command.emrserverless.openDashboardUI": "Open Dashboard UI...",
```

Comment:
If this is visiting the AWS console can it be named `Open AWS Console` or `Open Dashboard in AWS Console`, for consistency with existing actions: https://github.com/aws/aws-toolkit-vscode/blob/8428a21785c0d74b1a2f660d5b2a8347509e25b4/src/shared/ui/buttons.ts#L16

Don't think "UI" needs to be mentioned.

Reply:
This is slightly different. Inside the EMR Studio console, you have the option to open the relevant application UI for a specific job. For Spark jobs, this is the Spark UI or Spark History Server (see screenshot). This is an off-console UI (for which the API is `getDashboardForJobRun`, hence my usage of Dashboard). "Application UI" may be better as it's more consistent with the console.

![image](https://github.com/aws/aws-toolkit-vscode/assets/1512/e3c32b86-279c-4584-a115-ddd67c6754bc)